Functions.Templates/Templates/KafkaTrigger-CSharp/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+    "defaultFunctionName": "KafkaTrigger",
+    "description": "$KafkaTrigger_description",
+    "name": "Kafka trigger",
+    "language": "C#",
+    "category": [
+        "$temp_category_core",
+        "$temp_category_dataProcessing"
+    ],
+    "categoryStyle": "Kafka",
```

Comment:
There is  an invalid value `"categoryStyle": "Kafka",`. No such category.

Reply:
Done

Reply:
I made it as "other" It it what you expected?

Reply:
Yes.